surround/tests/cli/project_cli/Test_surround_init_lint.py

```diff
@@ -0,0 +1,117 @@
+import unittest
+import subprocess
+import shutil
+import os
+class MyTestCase(unittest.TestCase):
+#validating the surround init command line functionality
+#verfiying the output after the surround init command
+    def check_project_name(self):
+        self.test_surround_init()#for checking the Name of project input
+        self.test_surround_init_lowercase('tempor','temporary') #for testing the lowercase for project name-successfuly created
+        self.test_surround_init_lowercase_number('tempor123','temporary')#for testing the project name with lowercase and numbers
+        self.test_surround_init_lowercase_underscore('temporary_','temporary')#for testing the project name with lowercase & Underscore
+        self.test_surround_init_lowercase_symbol('temporary*#','temporary')#for testing the project name with lowercase&symbols
+        self.test_surround_init_onlysymbol(',#*@$','temporary')#for testing the project name with symbols
+        self.test_total_surround_files()#for validating total no of files inside the surround environment
+        self.test_created_surround_files()#for validating the created file names of surround project to the actual file name
+        self.test_surround_uppercase()
+
+
+    def test_surround_init(self):
+        process1= subprocess.run(["surround", "init"], encoding='utf-8', stdout=subprocess.PIPE,stdin=subprocess.PIPE)
+        print("Output captured:", process1.stdout)
+        self.assertIn(process1.stdout,"Name of project: ")
+
+    def test_surround_init_lowercase(self,name='tempor',description='temporary'):
+        process = subprocess.run(["surround", "init",'-p',name,'-d',description,'-w','no'], encoding='utf-8',
+                                 stdout=subprocess.PIPE,stdin=subprocess.PIPE)
+        print("Output captured:", process.stdout)
+        self.assertIn(process.stdout, "info: project created at C:\\Users\\sunda\\Surround_Test\\tempor\n")
```

Comment:
Using an absolute path will not work on machines other than your own
If I run this test, it will fail since my username is not ``sunda``